src/args/make-derivation/default.nix

```diff
@@ -99,7 +99,7 @@ builtins.derivation (env' // {
     scriptName="$1"
     shift 1
 
-    if test -f ''${BASH_SOURCE%/*}/README.md; then
+    if test $# -ge 1 && test -f ''${BASH_SOURCE%/*}/README.md; then
       case $1 in
```

Comment:
does this solve the problem?

```suggestion
      case "${1:-}" in
```

it also hardens the script against command injections (by quoting the var expansion)

Reply:
It did! Thx!